package.json

```diff
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "lint": "eslint .",
+    "pretest": "node test/helpers/pretest.js",
```

Comment:
interesting ! , I am going to use this for scenario specific setup, the pretest.js just cleans a sandbox folder and creates a fresh folder. It is not scenario specific